ca_description/urdf/sensors/cliff_sensors_gazebo.xacro

```diff
@@ -7,6 +7,7 @@
   <xacro:property name="max_range_m" value="0.04"/>
 
   <xacro:property name="rate_hz" value="20.0"/>
+  <xacro:property name="min_cliff_value" value="0.035"/>
```

Comment:
Something better:

```
<xacro:property name="min_cliff_value_cm" value="3.5"/>
```

And then create a property in `common_properties.xacro` called `cm_2_m` and use it here to convert to meters.